### DISCUSSION TITLE
Qnap Questions

### DISCUSSION THREAD

**Discussion by CluelessDocker** [itzg/docker-minecraft-server#1702] [Q&A]
I'm working with a qnas and attempting to edit the ops.json but every time a start the docker it just resets ops.json and i can't get ops on my server

**Comment by itzg** [comment]
Can you provide your container config? In any case, rather than edit the file directly you may just want to let the container manage it.

**Reply by CluelessDocker** [reply]
thanks for the quick response, sorry I was not so quick

ill try that
